[VSIX] The TemplatedControl Item Template does not contain the required partial modifier

I am following [these instructions](https://platform.uno/docs/articles/guides/how-to-create-control-libraries.html) to create a control library.

I added a Custom Control and the generated class looks like this:

    public sealed class CustomControl1 : Control
	
which causes this error:

> error CS0260: Missing partial modifier on declaration of type 'CustomControl1'; another partial declaration of this type exists


Other than `Class1` which is supplied by default when the project is created, there are no other controls or classes in the project.

Adding the `partial` modifier allows the project to build.


Interesting to note:  The error message does not appear in the VS Error List window.  I had to go to compliler output window to find it.  I don't recall ever getting an error in VS that does not show up in the error list window.

## Notes for contributors

- Find the `UnoItemTemplates` folder and find `CustomControl.cs` files there (there is one for UWP and one for WinUI)
- Add `partial` modifier after the `sealed` keyword in both files
- Celebrate 🎉

--- a/src/UnoItemTemplate/TemplatedControl.UWP/CustomControl.cs
+++ b/src/UnoItemTemplate/TemplatedControl.UWP/CustomControl.cs
@@ -14,7 +14,7 @@ using Windows.UI.Xaml.Media;
 
 namespace $rootnamespace$
 {
-    public sealed class $safeitemname$ : Control
+    public sealed partial class $safeitemname$ : Control
     {
         public $safeitemname$()
         {

--- a/src/UnoItemTemplate/TemplatedControl.WinUI/CustomControl.cs
+++ b/src/UnoItemTemplate/TemplatedControl.WinUI/CustomControl.cs
@@ -14,7 +14,7 @@ using Microsoft.UI.Xaml.Media;
 
 namespace $rootnamespace$
 {
-    public sealed class $safeitemname$ : Control
+    public sealed partial class $safeitemname$ : Control
     {
         public $safeitemname$()
         {

